tests/integration-tests/tests/efa/test_efa.py

```diff
@@ -197,8 +202,20 @@ def _test_osu_benchmarks_collective(
 
 
 def _test_osu_benchmarks_multiple_bandwidth(
-    remote_command_executor, scheduler_commands, test_datadir, slots_per_instance, partition=None
+    instance, remote_command_executor, scheduler_commands, test_datadir, slots_per_instance, partition=None
 ):
+    # Expected bandwidth with 4 NICS:
```

Comment:
these notes are valid for p4d instances only, let's highlight it or move these comments into the dict:

```python
instance_bandwidth_dict = {
        # Expected bandwidth for p4d and p4de (4 * 100 Gbps NICS -> declared NetworkPerformance 400 Gbps):
        # OMPI 4.1.0: ~330Gbps = 41250MB/s with Placement Group
        # OMPI 4.1.0: ~252Gbps = 31550MB/s without Placement Group
        # OMPI 4.0.5: ~95Gbps = 11875MB/s with Placement Group
        "p4d.24xlarge": 30000,  # Equivalent to a theoretical maximum of a single 240Gbps card
        "p4de.24xlarge": 30000,  # Equivalent to a theoretical maximum of a single 240Gbps card
        # # Expected bandwidth for hpc6id (2 up to 170 Gbps NICS -> declared NetworkPerformance 200 Gbps)
        "hpc6id.32xlarge": 23500,  # Equivalent to a theoretical maximum of a single 188Gbps card
    }
```

Reply:
good point, done